py3status/modules/whatismyip.py

```diff
@@ -13,14 +14,13 @@
         (default 'ip')
     negative_cache_timeout: how often to check again when offline (default 2)
     timeout: how long before deciding we're offline (default 5)
-    url: change IP check url (must output a plain text IP address)
-        (default 'http://ultrabug.fr/py3status/whatismyip')
     url_geo: IP to check for geo location (must output json)
-        (default 'http://ip-api.com/json')
+        (default 'https://freegeoip.net/json')
```

Comment:
When I'm on VPN, half of the time this address does not resolve to IP and so this module reports me as offline. The new default is always resolving fine.